xml/abstract_commands.xml

```diff
@@ -94,4 +94,82 @@
         <field name="0" bits="23:0" />
     </register>
 
+    <register name="Access Memory">
+        \begin{steps}{This command gives the debugger access to memory
+            locations, including memory-mapped registers. It performs the
+            following sequence of operations:}
+        \item Copy data from the memory location specified in {\tt arg1} into the
+            {\tt arg0} portion of {\tt data}, if \Fwrite is clear and \Ftransfer is set.
+        \item Copy data from the {\tt arg0} portion of {\tt data} into the
+            memory location specified in {\tt arg1}, if \Fwrite is set and \Ftransfer is set.
+        \item Increment {\tt arg1}, if \Fpostincrement is set.
```

Comment:
should we name these bits something less generic, like `amapostincrement` (abstract memory access post increement) .To avoid confusion with `sbaincrement` bits or the like.

Reply:
Yes. Done.